src/Calculator/Converters/BitFlipAutomationNameConverter.cpp

```diff
@@ -42,18 +42,19 @@ Object ^ BitFlipAutomationNameConverter::Convert(_In_ Object ^ value, TypeName t
         unsigned int binaryLength = 0;
 
         // remove all the characters except 0 and 1 from the array.
-        for
-            each(wchar_t bit in binaryDisplay)
+        // clang-format off
```

Comment:
What was the problem here? I imagine that's some kind of foreach macro? If that's it, `clang-format` has `ForEachMacros` or it can be changed to C++'s ranged-for

Reply:
@janisozaur the problem was that `clang-format` moved `each` on the next line, as you can see from the diff. And it was not a valid construction. The build was failing here. 
If you see how to fix it better feel free to push commit in my PR. :)
I would check later `ForEachMacros`, thanks.

Reply:
@janisozaur probalby if I add this with space
```
ForEachMacros:   
  - for each
  - foreach
```
to the list, it should work. I will check.